templates/openssl.cnf.erb

```diff
@@ -25,6 +25,7 @@ default_crl_days= 30
 default_md	= md5
 preserve	= no
 policy		= policy_anything
+unique_subject	= no
```

Comment:
should this be configurable?

Reply:
I don't see much point in making it configurable. According to [the docs](https://manpages.debian.org/stretch/openssl/ca.1ssl.en.html#CONFIGURATION_FILE_OPTIONS):

> The default value is yes, to be compatible with older (pre 0.9.8) versions of OpenSSL.

OpenSSL 0.9.8 is old enough to not be a concern, any more, I think. 1.0.0 was released in 2010.